test/scripts/validate-param-sys.py

```diff
@@ -0,0 +1,132 @@
+########################################################################
+# This script is designed to function with OpenC3 COSMOS's Script Runner
+########################################################################
+from openc3.script import *
+import struct
+
+# Clear counts and data products from prior tests:
+cmd("Linux_Example Ccsds_Command_Depacketizer_Instance-Reset_Counts")
+cmd("Linux_Example Command_Router_Instance-Reset_Data_Products")
+wait(1.0)
```

Comment:
Can we wrap some of this up into a file of its own and call it `test_setup` or something? I have a feeling we will want to be doing stuff like this to preamble each test we write.

Reply:
Addressed in [2b86ad4](https://github.com/lasp/adamant_example/pull/33/commits/2b86ad426e8c0c7903519452764c5bdb6c5bc497).